index.js

```diff
@@ -12,6 +13,10 @@ module.exports = function (numbers, opts) {
 	var finiteNumbers = numbers.filter(isFinite);
 	var min = opts.min || Math.min.apply(null, finiteNumbers);
 	var max = opts.max || Math.max.apply(null, finiteNumbers);
+	var style = opts.style || '';
```

Comment:
No need to create a separate `style` variable. Just use `opts.style === 'fire'`.